ingest/src/commands/embed.ts

```diff
@@ -18,45 +20,20 @@ const commandModule: CommandModule<unknown, EmbeddedContentCommandArgs> = {
     return args.string("since").string("source").demandOption("since");
   },
   async handler({ since, source }) {
```

Comment:
can you do some light data validation on the `since` param? 

something like:

```ts
if(!Date.parse(since)){
  throw new Error(`The value for 'since' (${since}) must be a valid JavaScript date string.`)
}

```

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse